tests/Tests/100__deploy/130__operators/130__rhods_operator/138__rhods_operator_oom_kill_verification.robot

```diff
@@ -0,0 +1,63 @@
+*** Settings ***
+Documentation       RHODS_OPERATOR_OOM_KILL_VERIFICATION
+...                 Verify rhods operator running without any issue
+...                 after createing multiple dummy namespace in openshift
+...
+...                 = Variables =
+...                 | Namespace    | Required |    RHODS operator namespace|
+...                 | Number    | Required |    Number of namespace to be created |
+
+Library             OperatingSystem
+Library             SeleniumLibrary
+Resource            ../../../../Resources/Page/ODH/JupyterHub/ODHJupyterhub.resource
+Resource            ../../../../Resources/Page/OCPDashboard/OCPDashboard.resource
+Resource            ../../../../Resources/Common.robot
+
+
+*** Variables ***
+${NAMESPACE}    redhat-ods-operator
+${NUMBER}       1500
+
+
+*** Test Cases ***
+Verify RHODS Operator OOM Kill Behaviour
+    [Documentation]    Create multiple namespace and verify
+    ...    if the rhods operator pod is running without any
+    ...    issue and perfrom some basic validation with RHODS
+    [Tags]    ODS-1091    Tier3
+    ${dfeault_np_count}    Run    oc get namespace | wc -l
+    Create Namspace In Opneshift
+    Verify Operator Pod Status    ${NAMESPACE}    name=rhods-operator
+    Basic Dashboard Test Verfication
+    Delete Namespace From Openshift
+    ${new__np_count}    Run    oc get namespace | wc -l
+    Should Be True    ${dfeault_np_count} == ${new__np_count}    All the dummy namespace created is not deleted
+
+
+*** Keywords ***
+Create Namspace In Opneshift
```

Comment:
typo in keyword name, should be `Create Namespace In Openshift`

Reply:
Done